requirements.txt

```diff
@@ -1,6 +1,59 @@
-openai==1.55.3
+aiofiles==24.1.0
+aiohappyeyeballs==2.4.4
+aiohttp==3.11.10
+aiosignal==1.3.2
+annotated-types==0.7.0
+anyio==4.7.0
+attrs==24.3.0
+certifi==2024.12.14
+cffi==1.17.1
+charset-normalizer==3.4.0
+cryptography==44.0.0
+dataclasses-json==0.6.7
+Deprecated==1.2.15
+distro==1.9.0
+frozenlist==1.5.0
+h11==0.14.0
+httpcore==1.0.7
 httpx==0.27.2
-langchain==0.1.12
-langchain-openai==0.0.8
-PyGithub==2.2.0
-pydantic==2.6.4 
\ No newline at end of file
+httpx-sse==0.4.0
+idna==3.10
+jiter==0.8.2
+jsonpatch==1.33
+jsonpointer==3.0.0
+langchain==0.3.12
+langchain-community==0.3.12
+langchain-core==0.3.25
+langchain-openai==0.2.12
+langchain-text-splitters==0.3.3
+langsmith==0.2.3
+marshmallow==3.23.1
+multidict==6.1.0
+mypy-extensions==1.0.0
+numpy==1.26.4
+openai==1.55.3
+orjson==3.10.12
+packaging==23.2
+propcache==0.2.1
```

Comment:
🔄 If any packages are for development/testing, consider moving them to a separate `requirements-dev.txt`.